tests/test_inject_ib_uuids.py

```diff
@@ -82,7 +82,6 @@ def test_inject_ib_ids_input_and_cta(
 
     with open(fp2, 'r') as f:
         result = f.read()
-        print(result)
```

Comment:
Minor: Probably this is a remnant of debugging that can be removed :man_shrugging:

Reply:
Oh -- wow, I read this diff the other way despite it being clearly "red" for delete vs add. Ignore :joy: .